rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -489,14 +512,14 @@ public boolean preemptTask(String jobId, String taskName, int restartDelay) thro
     }
 
     @Override
-    public boolean removeJob(JobId arg0) throws NotConnectedException, UnknownJobException,
-            PermissionException {
+    public boolean removeJob(JobId arg0)
+            throws NotConnectedException, UnknownJobException, PermissionException {
         throw new UnsupportedOperationException();
```

Comment:
I know it's not part of this commit, but maybe you could use it to implement this method why you're there ?